src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Extensions/SyntaxGeneratorExtensions_Negate.cs

```diff
@@ -130,48 +127,44 @@ internal static partial class SyntaxGeneratorExtensions
             }
 
             if (!s_negatedBinaryMap.TryGetValue(binaryOperation.OperatorKind, out var negatedKind))
-            {
                 return generator.LogicalNotExpression(expressionNode);
-            }
-            else
+
+            var negateOperands = false;
```

Comment:
Why did this one not get the switch expression treatment? Or even just assign it to a pattern?

Reply:
I'll change :-)